articles/app-service/monitor-instances-health-check.md

```diff
@@ -136,6 +136,11 @@ If you restart the instance and the restart process fails, you will then be give
 
 Windows applications will also have the option to view processes via the Process Explorer.  This gives you further insight on the instance's processes including thread count, private memory, and total CPU time.
 
+## Diagnostic information collection
+For Windows applications, you have the option to collect diagnostic information in the Health Check tab.  When an instance becomes unhealthy, you can collect a memory dump using a storage account to view more information on why your instance may be unhealthy.  Enabling this option will change exiting auto-heal configurations.  
```

Comment:
For Windows applications, you have the option to collect diagnostic information in the Health Check tab. Enabling diagnostic collection will add an auto-heal rule that creates memory dumps for unhealthy instances and saves it to a storage account. Enabling this option will change auto-heal configurations, so if there are existing auto-heal rules, we recommend setting this up through App Service diagnostics.